mongo/metadata.csv

```diff
@@ -225,14 +225,14 @@ mongodb.usage.update.time,gauge,,microsecond,,Total time spent performing update
 mongodb.usage.writeLock.countps,rate,,lock,,Number of write locks per second,0,mongodb,
 mongodb.usage.writeLock.count,gauge,,lock,,Number of write locks since server start (deprecated),0,mongodb,
 mongodb.usage.writeLock.time,gauge,,microsecond,,Total time spent performing write locks in microseconds,-1,mongodb,
-mongodb.tcmalloc.generic.current_allocated_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.generic.heap_size,gauge,,unit,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.aggressive_memory_decommit,gauge,,unit,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.central_cache_free_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.current_total_thread_cache_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.max_total_thread_cache_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.pageheap_free_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.pageheap_unmapped_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.thread_cache_free_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.transfer_cache_free_bytes,gauge,,byte,,,0,mongodb,
-mongodb.tcmalloc.tcmalloc.spinlock_total_delay_ns,gauge,,nanosecond,,,0,mongodb,
+mongodb.tcmalloc.generic.current_allocated_bytes,gauge,,byte,,Number of bytes used by the application.,0,mongodb,
+mongodb.tcmalloc.generic.heap_size,gauge,,byte,,Bytes of system memory reserved by TCMalloc.,0,mongodb,
+mongodb.tcmalloc.tcmalloc.aggressive_memory_decommit,gauge,,unit,,Status of aggressive memory decommit mode.,0,mongodb,
+mongodb.tcmalloc.tcmalloc.central_cache_free_bytes,gauge,,byte,,Number of free bytes in the central cache.,0,mongodb,
+mongodb.tcmalloc.tcmalloc.current_total_thread_cache_bytes,gauge,,byte,,Number of bytes used across all thread caches.,0,mongodb,
+mongodb.tcmalloc.tcmalloc.max_total_thread_cache_bytes,gauge,,byte,,Upper limit on total number of bytes stored across all per-thread caches.,0,mongodb,
+mongodb.tcmalloc.tcmalloc.pageheap_free_bytes,gauge,,byte,,Number of bytes in free mapped pages in page heap.,0,mongodb,
+mongodb.tcmalloc.tcmalloc.pageheap_unmapped_bytes,gauge,,byte,,Number of bytes in free unmapped pages in page heap.,0,mongodb,
+mongodb.tcmalloc.tcmalloc.thread_cache_free_bytes,gauge,,byte,,Number of free bytes in thread caches.,0,mongodb,
+mongodb.tcmalloc.tcmalloc.transfer_cache_free_bytes,gauge,,byte,,Number of free bytes that are waiting to be transfered between the central cache and a thread cache.,0,mongodb,
+mongodb.tcmalloc.tcmalloc.spinlock_total_delay_ns,gauge,,nanosecond,,Spinlock delay time.,-1,mongodb,
```

Comment:
Seems that the convention is to not put a `.` at the end of the description.

Reply:
Actually, I have doubt, seems not always the case.

@ruthnaebeck Any suggestion on this ? :)

Reply:
The convention depends a lot on the integration. Some of them have `.` to all the metrics, some don't, I don't know which one is the best one. For this integration, some descriptions have `.` and some don't

Reply:
In general, it's actually a sentence, use `.` otherwise no.